dist/README.md

```diff
@@ -154,7 +154,35 @@ The extra variables defined in `persist/grist-env` will only be
 applied to the Grist Docker Compose service. In particular, they will
 not affect Traefik's, Dex's, or Authelia's environment.
 
-# Updating Grist
+## Advanced profile
+
+For advanced use cases where the authentication provided by Dex and
+Authelia is not adequate, it is possible to enable an advanced Docker
+Compose profile. This profile will only configure Traefik and start
+with a completely blank Grist configuration. To generate an advanced
+configuration, from a clean slate do
+
+```sh
+COMPOSE_PROFILES=advanced \
+DEFAULT_EMAIL=gristadmin@example.com \
+GRIST_DOMAIN=grist.example.com \
+./bin/bootstrap-environment
+```
+
+All of the Grist configuration must now be provided via the
+supplemental `persist/grist-env` file described in the previous
+section. At a bare minimmum, this requires setting `APP_HOME_URL`. We
+also recommend setting the following variables:
+
+* `GRIST_DEFAULT_EMAIL`
+* `GRIST_SINGLE_ORG`
```

Comment:
Mmm `GRIST_SINGLE_ORG` is a bit incidental there, a legacy of starting with `grist-omnibus` that just made this opinionated choice for people. Trying to remember why. It might have been the easiest way to lock down access. With multi-org, by default anonymous folks can try things out. But now there's a flag to turn that off. I think that's a better thing to recommend considering now: `GRIST_ANON_PLAYGROUND=false`.

Reply:
I modified the recommended variables to your suggestion.